modules/fedimint-mint-client/src/backup/recovery.rs

```diff
@@ -246,20 +247,32 @@ impl MintRestoreInProgressState {
         decoders: ModuleDecoderRegistry,
         secret: DerivableSecret,
     ) -> Self {
+        const PROGRESS_SNAPSHOT_BLOCKS: u64 = 50;
+
         assert_eq!(secret.level(), 2);
 
-        info!(target: LOG_CLIENT_RECOVERY_MINT, "Processing block {}", self.next_epoch);
+        let block_range = self.next_epoch
+            ..cmp::min(
+                self.next_epoch.wrapping_add(PROGRESS_SNAPSHOT_BLOCKS),
+                self.end_epoch,
+            );
```

Comment:
This is kinda ugly syntax :/

Reply:
In what sense?

Reply:
A range that's defined over multiple lines seems a bit unusual, but it's only a nit.